core/performance_test/peakflops/03_Cabana_peakflops.cpp

```diff
@@ -299,10 +304,14 @@ void run()
     std::cout<<dc1<<" clocks 1"<<std::endl;
     std::cout<<flops/dc1<<" flops/clock 1\n";
     std::cout << std::endl;
-    std::cout << "Slice Single Index" << std::endl;
+
+    double flops_clock = flops / dc1;
```

Comment:
What about checking the performance of the other 2 ways of accessing the data?

Reply:
Do we want to add that?

I had originally left it out because:

1. It doesn't test any Cabana functionality
2. If they don't work, it's basically impossible for the existing test to pass (so it's somewhat subsumed)

I'm happy to add it though!

Reply:
I would argue we don't test any Cabana functionality in the benchmark if we don't check the other 2. Shouldn't we expect peak flops when using slice with `access`?

Reply:
Having thought about it a bit more, I agree (and in fact my comment above wasn't even strictly true). I'll add it now -- good catch.